test/bun.js/websocket-server.test.ts

```diff
@@ -604,7 +603,7 @@ describe("websocket server", () => {
           ws.send("hello world");
         },
         message(ws, msg) {
-          ws.cork(() => {
+          ws.cork!(() => {
```

Comment:
is this missing from the types? `cork` is never undefined

Reply:
Fixed